tools/dxexp/dxexp.cpp

```diff
@@ -147,18 +147,12 @@ typedef struct D3D12_FEATURE_DATA_D3D12_OPTIONS7 {
 #if WDK_NTDDI_VERSION < NTDDI_WIN10_FE
 #define D3D12_FEATURE_D3D12_OPTIONS9 ((D3D12_FEATURE)37)
 
-typedef enum D3D12_WAVE_MMA_TIER {
-  D3D12_WAVE_MMA_TIER_NOT_SUPPORTED = 0,
-  D3D12_WAVE_MMA_TIER_1_0 = 10
-} D3D12_WAVE_MMA_TIER;
-
 typedef struct D3D12_FEATURE_DATA_D3D12_OPTIONS9 {
   BOOL MeshShaderPipelineStatsSupported;
   BOOL MeshShaderSupportsFullRangeRenderTargetArrayIndex;
   BOOL AtomicInt64OnTypedResourceSupported;
   BOOL AtomicInt64OnGroupSharedSupported;
   BOOL DerivativesInMeshAndAmplificationShadersSupported;
-  D3D12_WAVE_MMA_TIER WaveMMATier;
 } D3D12_FEATURE_DATA_D3D12_OPTIONS9;
```

Comment:
This might need to stay in, it is already shipped:
https://learn.microsoft.com/en-us/windows/win32/api/d3d12/ns-d3d12-d3d12_feature_data_d3d12_options9

Reply:
Presumably these structs should match the ones in [d3d12.h](https://github.com/microsoft/DirectX-Headers/blob/9be295b3b81ce1d0ff2b44f18d0eb86ea54c5122/include/directx/d3d12.h#L2757)

Reply:
Updated.